config/default.yml

```
@@ -354,6 +354,12 @@ Rails/Pick:
   Safe: false
   VersionAdded: '2.6'
 
+Rails/PluckInWhere:
+  Description: 'Use `select` instead of `pluck` in `where` query methods.'
+  Enabled: true
```

Can you update `Enabled: true` to `Enabled: 'pending'`?

```suggestion
  Enabled: 'pending'
```